### PR TITLE
Add per-plan operations to CLI and enable passing of envvars to start command

### PR DESCRIPTION
**Pull request by mrbrowning**

### CODE REVIEW

**Comment by mrbrowning** [comment]
@mesosphere-ci retest this please

**Comment by mrbrowning** [comment]
@mesosphere-ci retest this please

**Comment by mrbrowning** [comment]
@mesosphere-ci retest this please

**Comment by susanxhuynh** [comment]
Do we have tests for the CLI? Or just for the REST endpoints?

**Comment by mrbrowning** [comment]
No tests for the CLI, just manual verification at the moment.

File: cli/commands.go [hunk 1]
```diff
@@ -241,18 +304,33 @@ func HandlePlanSection(app *kingpin.Application) {
...
-	plan.Command("interrupt", "Interrupt the current InProgress operation").Action(cmd.RunInterrupt)
+	show := plan.Command("show", "Display the deploy plan or the plan with the provided name").Action(cmd.RunShow)
+	show.Arg("plan", "Name of the plan to show").StringVar(&cmd.PlanName)
 
-	force := plan.Command("force", "Force the current operation to complete").Action(cmd.RunForce)
-	force.Arg("phase", "UUID of the Phase containing the provided Step").Required().StringVar(&cmd.PhaseId)
-	force.Arg("step", "UUID of Step to be restarted").Required().StringVar(&cmd.StepId)
+	start := plan.Command("start", "Start the plan with the provided name, with optional envvars to supply to task").Action(cmd.RunStart)
+	start.Arg("plan", "Name of the plan to start").Required().StringVar(&cmd.PlanName)
+	start.Arg("params", "Comma-separated list of VAR=value pairs").Required().StringVar(&cmd.Parameters)
```
Comment (susanxhuynh) [inline cli/commands.go]: Should "params" be optional rather than required?

Comment (mrbrowning) [inline cli/commands.go]: Yeah, it should be. I'll fix.

**Pull request merged**